modules/ocf/templates/kubectl/kubectl.conf.erb

```diff
@@ -0,0 +1,20 @@
+users:
+- name: ocfstaffer
+  user:
+    exec:
+      command: "/opt/share/utils/sbin/gen-kubernetes-cert"
+      apiVersion: "client.authentication.k8s.io/v1beta1"
+
+clusters:
+- name: prod
+  cluster:
+    certificate-authority-data: <%= @cluster_cert_base64.gsub(/\n/, "") %>
```

Comment:
ruby has a `delete` function you can use for this. imo more readable than gsub.

Reply:
coolio